src/cascadia/TerminalApp/TerminalPage.h

```diff
@@ -76,6 +76,7 @@ namespace winrt::TerminalApp::implementation
         TerminalApp::TabRowControl _tabRow{ nullptr };
         Windows::UI::Xaml::Controls::Grid _tabContent{ nullptr };
         Microsoft::UI::Xaml::Controls::SplitButton _newTabButton{ nullptr };
+        TerminalApp::CommandPalette _commandPalette{ nullptr };
```

Comment:
Do we really need to stash this value? I'm not sure this is being used anywhere...